src/client/view/components/TkComments.vue

```diff
@@ -161,4 +161,31 @@ export default {
   cursor: pointer;
   color: #409eff;
 }
+div.code-toolbar {
```

Comment:
所有 css 选择器应以 `.tk-` 开头，或添加 `.twikoo` 限定，避免影响评论区之外的元素。

```suggestion
.twikoo div.code-toolbar {
```

（下同）